mark-kavdi/server.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+// DONE: Install and require the NPM Postgres package 'pg' into your server.js, and ensure that it is then listed as a dependency in your package.json
+const fs = require('fs');
+const express = require('express');
+const PG = require('pg');
+
+// REVIEW: Require in body-parser for post requests in our server. If you want to know more about what this does, read the docs!
+const bodyParser = require('body-parser');
+const PORT = process.env.PORT || 3000;
+const app = express();
+
+// DONE: Complete the connection string for the url that will connect to your local postgres database
+// Windows and Linux users; You should have retained the user/pw from the pre-work for this course.
+// Your url may require that it's composed of additional information including user and password
+// const conString = 'postgres://USER:PASSWORD@HOST:PORT/DBNAME';
+const conString = 'postgres://localhost:5432/articles';
```

Comment:
`articles` is the name of the table that we are creating, not the database. The database should be `kilovolt`.